src/views/blocks/tickets.php

```diff
@@ -73,56 +76,64 @@
 	data-provider-id="<?php echo esc_attr( $provider->orm_provider ); ?>"
 	novalidate
 >
-	<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title"><?php echo esc_html( tribe_get_ticket_label_plural( 'event-tickets' ) ); ?></h2>
+	<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title">
+		<?php echo esc_html( tribe_get_ticket_label_plural( 'event-tickets' ) ); ?>
+	</h2>
 	<input type="hidden" name="tribe_tickets_saving_attendees" value="1"/>
 	<input type="hidden" name="tribe_tickets_ar" value="1"/>
 	<input type="hidden" name="tribe_tickets_ar_data" value="" id="tribe_tickets_block_ar_data"/>
-	<?php $this->template(
+	<?php
+	$this->template(
 		'components/notice',
 		[
-			'id' => 'tribe-tickets__notice__tickets-in-cart',
-			'notice_classes' => [
+			'id'              => 'tribe-tickets__notice__tickets-in-cart',
+			'notice_classes'  => [
 				'tribe-tickets__notice--barred',
 				'tribe-tickets__notice--barred-left',
 			],
 
 			'content_classes' => [
 				'tribe-common-b3',
 			],
-			'content' => __( 'The numbers below include tickets for this event already in your cart. Clicking "Get Tickets" will allow you to edit any existing attendee information as well as change ticket quantities.', 'event-tickets' )
+			'content'         => __( 'The numbers below include tickets for this event already in your cart. Clicking "Get Tickets" will allow you to edit any existing attendee information as well as change ticket quantities.', 'event-tickets' ),
 		]
-	); ?>
+	);
+	?>
 
 	<?php $this->template( 'blocks/tickets/commerce/fields', [ 'provider' => $provider, 'provider_id' => $provider_id ] ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: e41c99b19060b96d3a1da6b8299b0adb</sub>